packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -1229,10 +1287,57 @@ class MemoryBodyState extends State<MemoryBody>
   void showLegend(BuildContext context) {
     final RenderBox box = legendKey.currentContext.findRenderObject();
 
+    final colorScheme = Theme.of(context).colorScheme;
+    final legendHeading = colorScheme.hoverTextStyle;
+
     // Global position.
     final position = box.localToGlobal(Offset.zero);
 
-    final legendHeading = Theme.of(context).textTheme.subtitle2;
+    final legendRows = <Widget>[];
+
+    final events = eventLegend(colorScheme.isLight);
+    legendRows.add(Container(
+      padding: const EdgeInsets.fromLTRB(5, 0, 0, 4),
+      child: Text('Events Legend', style: legendHeading),
+    ));
+
+    var iterator = events.entries.iterator;
+    while (iterator.moveNext()) {
+      final leftEntry = iterator.current;
+      final rightEntry = iterator.moveNext() ? iterator.current : null;
+      legendRows.add(legendRow(entry1: leftEntry, entry2: rightEntry));
+    }
+
+    final vms = vmLegend();
+    legendRows.add(
+      Container(
+        padding: const EdgeInsets.fromLTRB(5, 0, 0, 4),
```

Comment:
this padding is used in multiple places - pull it out into a const that can be used everywhere this is repeated

Reply:
/// Padding for each title in the legend.
  static const _legendTitlePadding = EdgeInsets.fromLTRB(5, 0, 0, 4);